verification/curator-service/api/src/controllers/cases.ts

```diff
@@ -10,6 +10,9 @@ import crypto from 'crypto';
 //   https://github.com/globaldothealth/list/issues/961.
 axios.defaults.timeout = 0;
 
+// The query (URL) when the line list first loads
+const defaultQuery = '/cases/?limit=50&page=1&count_limit=10000&sort_by=default&order=ascending';
+
```

Comment:
this seems brittle as the UI and API need to agree on this query, I can imagine us changing it in the UI and forgetting to update this line. Perhaps the UI can send a sentinel query, e.g `?limit=default&page=default&…` and the API tests for whether the qs contain any non-default fields?

Alternatively, the UI _doesn't_ request any query parameters by default, and the default value in the API is the one the UI expects. API customers can still request other limits/pages/counts etc by explicitly setting them, at which point we log their query.

Reply:
I agree. Thanks for providing those suggestions; I think the second one makes the most sense as it keeps the responsibility of the definition of "default" internal. Going to pursue that.